env.example

```diff
@@ -49,3 +49,10 @@ DJANGO_SERVER_EMAIL=
 # Sentry
 # Only needed if you want to send errors to Sentry
 DJANGO_SENTRY_DSN=
+
+
+# GUS BUS
+export METACI_RELEASE_WEBHOOK_URL=http://0.0.0.0:8001
+export METACI_RELEASE_WEBHOOK_AUTH_KEY=123645
+export METACI_START_STOP_WEBHOOK=True
+export GUS_BUS_OWNER_ID="005B00000000000000" # must edit yourself
```

Comment:
Let's keep these commented out in env.example. If someone else is trying to figure out how to run MetaCI locally I don't want them to get hung up because they aren't using the GUS integration.